packages/sanity/src/desk/deskTool.ts

```diff
@@ -1,12 +1,34 @@
 import {MasterDetailIcon} from '@sanity/icons'
 import {lazy} from 'react'
-import {createPlugin} from '../config'
+import {
+  DeleteAction,
+  DiscardChangesAction,
+  DuplicateAction,
+  HistoryRestoreAction,
+  PublishAction,
+  UnpublishAction,
+} from './documentActions'
+import {LiveEditBadge} from './documentBadges'
 import {getIntentState} from './getIntentState'
 import {router} from './router'
 import {DeskToolOptions} from './types'
+import {createPlugin} from 'sanity'
 
+/** @beta */
 export const deskTool = createPlugin<DeskToolOptions | void>((options) => ({
   name: '@sanity/desk-tool',
+  document: {
```

Comment:
DeskTool now creates its own actions, as mentioned in elsewhere right?

Reply:
Yes, exactly. This was part of the effort to remove `sanity/desk` dependencies in `sanity`. Before, these "default" actions were imported in the config creation.

But thinking of this now, this is still problematic. What if you have 2 desk tools? These will be duplicated. So need to fix that.